rook/scripts/autoconfig.sh

```diff
@@ -0,0 +1,160 @@
+#!/usr/bin/env bash
+
+# fn <address> <subnet>
+addressInSubnet() {
+  python3 -c "import ipaddress; exit(0) if ipaddress.ip_address('${1}') in ipaddress.ip_network('${2}') else exit(1)"
+}
+
+# fn <key> <addresses>
+updatePeers() {
+  peers="$(yq4 ".commons * .clusters.${cluster} | .networkPolicies.${1} | .[].cidr" "${CK8S_CONFIG_PATH}/rook/values.yaml")"
+
+  local -a update
+
+  for address in ${2}; do
+    for peer in ${peers}; do
+      if addressInSubnet "${address}" "${peer}"; then
+        update+=("${peer}")
+        continue 2
+      fi
+    done
+
+    update+=("${address}/32")
+  done
```

Comment:
I was unable to replicate the second issue, however I did find that I didn't skip /32 suffix peers so that's now fixed.

Reply:
> it will only keep subnets that seem to have something in them

Right, then I was misunderstanding the function.

> I was unable to replicate the second issue

Hmm, I must've mixed up `continue` with `break` when I read this the last time. Because I can see that it should work